cli/src/main/java/org/opentosca/toscana/cli/CliProperties.java

```diff
@@ -0,0 +1,101 @@
+package org.opentosca.toscana.cli;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import org.apache.commons.io.FileUtils;
+
+public class CliProperties {
+
+    public final String CLI_PROPS_ENDPOINT_KEY = "endpoint";
+    private final Properties properties = new Properties();
+    private String filePath;
+    private String dataPath = null;
+    private File file;
+    private String operatingSystem = null;
+
+    public CliProperties() {
+        setupPath();
+    }
+
+    /**
+     * Creates a cli.properties config file
+     */
+    private void createProperties() {
+        try {
+            FileWriter writer = new FileWriter(file);
+            properties.setProperty(CLI_PROPS_ENDPOINT_KEY, "http://127.0.0.1:8084/");
+            properties.store(writer, "Cli Settings");
+            writer.close();
+        } catch (IOException e) {
+            System.err.println("Something went wrong while trying to store the cli.properties config file");
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * Gets the API Url from the cli.properties config, if the config doesn't exist a default config gets created
+     *
+     * @return API Url
+     */
+    public String getApiUrl() {
+        String url = "";
+
+        try {
+            InputStream inputStream = new FileInputStream(file);
+            properties.load(inputStream);
+            inputStream.close();
+        } catch (IOException e) {
+            System.err.println("Something went wrong while trying to load the API Endpoint.");
+            e.printStackTrace();
+        }
+
+        url += properties.getProperty(CLI_PROPS_ENDPOINT_KEY);
+        return url;
+    }
+
+    /**
```

Comment:
Please check [how to contribute](https://github.com/StuPro-TOSCAna/TOSCAna/blob/master/CONTRIBUTING.md). You need to import the checkstyle and reformat the code. Javadoc comments should then be formatted without leading asterisks, like in the server module.